pkgs/top-level/python-packages.nix

```diff
@@ -4196,6 +4196,28 @@ in modules // {
       description = "pytest plugin with mechanisms for caching across test runs";
     };
   };
+  
+  pytestdjango = buildPythonPackage rec {
+    name = "pytest-django-${version}";
+    version = "2.9.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pytest-django/${name}.tar.gz";
+      sha256 = "1mmc7zsz3dlhs6sx4sppkj1vgshabi362r1a8b8wpj1qfximpqcb";
+    };
+    
+    # doing this to allow depending packages to find
+    # pytest's binaries
+    pytest = self.pytest;
```

Comment:
That is a good reason. But does that work like this? Could you show an expression how you depend on `pytestdjango`? As far as I know you can only refer `pytest` if you add it as an attribute to `passthru`. Right now it is an extra argument to `buildPythonPackage` which is not used and thus ignored.